frontend/src/components/UI/AddRecipeFab.vue

```diff
@@ -35,29 +34,29 @@
               :hint="$t('new-recipe.url-form-hint')"
               persistent-hint
             ></v-text-field>
-          </v-form>
 
-          <v-alert v-if="error" color="red" outlined type="success">
-            {{ $t("new-recipe.error-message") }}
-          </v-alert>
-        </v-card-text>
+            <v-alert v-if="error" color="red" outlined type="success">
+              {{ $t("new-recipe.error-message") }}
+            </v-alert>
+          </v-card-text>
 
-        <v-divider></v-divider>
+          <v-divider></v-divider>
 
-        <v-card-actions>
-          <v-spacer></v-spacer>
-          <v-btn color="grey" text @click="reset">
-            {{ $t("general.close") }}
-          </v-btn>
-          <v-btn
-            color="success"
-            text
-            @click="createRecipe"
-            :loading="processing"
-          >
-            {{ $t("general.submit") }}
-          </v-btn>
-        </v-card-actions>
+          <v-card-actions>
+            <v-spacer></v-spacer>
+            <v-btn color="grey" text @click="reset">
+              {{ $t("general.close") }}
+            </v-btn>
+            <v-btn
+              color="success"
+              text
+              @click.prevent="createRecipe"
```

Comment:
This click handler shouldn't be necessary. Instead of preventing the form's submit action from running only to run the same method it would normally run, the click handler should be removed entirely and replaced with `type="submit"`. This turns it into a native form submit button, utilising native browser functionality.

Reply:
Ah, you are right. I misremembered the requirements of the `.prevent` and where it was needed to keep the SPA from reloading. I've addressed this in #225. Thanks for the input!